.bingo/golangci-lint.mod

```diff
@@ -1,7 +1,7 @@
 module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
 
-go 1.22
+go 1.22.1
 
-toolchain go1.22.4
+toolchain go1.23.5
```

Comment:
Just noticed that while the Go version matches everywhere in the PR, we want 1.23.7 :)